tests/core_tests/test_context.py

```diff
@@ -74,16 +72,6 @@ def setUp(self):
         self.other_user_path = os.path.join(self.step_path, "user_login")
         self.add_production_path(self.other_user_path, self.other_user)
 
-        # adding shot path with alternate root 
```

Comment:
Moved this test in a separate class to the rest of the tests here could derive from TankTestSimple